backend/src/apiserver/server/job_server.go

```diff
@@ -42,6 +47,9 @@ func (s *JobServer) CreateJob(ctx context.Context, request *api.CreateJobRequest
 }
 
 func (s *JobServer) GetJob(ctx context.Context, request *api.GetJobRequest) (*api.Job, error) {
+	if common.IsMultiUserMode() == true {
```

Comment:
i think we just need to disable create/delete job api otherwise it will lead to UI navigation issues.

Reply:
Done.